test/helper.ts

```diff
@@ -80,4 +80,21 @@ module Lint.Test {
             assert(false, "expected " + stringifiedNeedle + " within " + stringifiedHaystack);
         }
     }
+
+    // assert that a failure array does not contain any error on a set of lines
+    export function assertNoFailuresWithin(fileName: string, failures: Lint.RuleFailure[], startLine: number, endLine: number) {
```

Comment:
The way we've actually been doing this is assert.lengthOf with actual failiures and however many failures there should be -- so that's the test you should add, rather than adding this code here.